test/support/helpers/request-field-validation-helper.ts

```diff
@@ -0,0 +1,3 @@
+export const prepareModifiedRequest = (requestIsAnArray: boolean, modifiedRequest) => {
```

Comment:
Type for midifiedREquest please.

Reply:
Added type `unknown`, this function should work with various types.

Reply:
Had to change from `unknown` to `any`. 
Having unknown started quite a lot of type checks and missmatch.